kernel/BT.c

```diff
@@ -398,7 +398,7 @@ static s32 BTHandleData(void *arg,void *buffer,u16 len)
 	}
 	else if(*(u8*)buffer == 0x22)	//acknowledge output report, return function result 
 	{
-		if(*((u8*)buffer+3) & 0x02)	//??message being acknowledged todo lucky all needed messages had 2 bit set

```

Comment:
What value does the 3nd party Wiimote deliver? Would it be sufficient to check for this?